tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/AssemblyInfoExceptionsStruct.cs

```diff
@@ -0,0 +1,15 @@
+// <copyright file="AssemblyInfoExceptionsStruct.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+using System;
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2;
+
+[DuckCopy]
+internal struct AssemblyInfoExceptionsStruct
```

Comment:
nit: don't need to add them in now, but I like when we include links to the github types these are duck typing, makes it easier to verify nullability annotations are correct etc + we'll need an explicit link for AOT in the (distant) future

Reply:
Oh man for AOT I probably will need to redo everything to split between versions, 2.x.0 is totally different to 3.[0-7] and different to 3.8.x . That means I'm 80% sure that this ducktype proxy maps with different types for different versions 😬